### ISSUE TITLE
elektra not compiling with gcc10

### ISSUE THREAD

**Issue by neheb** [openwrt/packages#12307]
```
[ 98%] Linking C executable ../bin/benchmark_large
/usr/bin/ld: CMakeFiles/benchmark_large.dir/large.c.o:(.bss+0x0): multiple definition of `large'; CMakeFiles/benchmark_large.dir/benchmarks.c.o:(.bss+0x0): first defined here
collect2: error: ld returned 1 exit status
```

This is the host build. Tested with:

```
make package/elektra/host/compile V=s
```

Maintainer: @haraldg
Environment: fedora 32, gcc10

**Comment by haraldg** [comment]
Thanks, I will have a look.

**Comment by haraldg** [comment]
@markus2330 Is this issue fixed in any released version?

(We probably should disable building benchmarks, but might be a nice excuse to update the package anyway.)

**Comment by mpranj** [comment]
@haraldg we have fixed this problem, and the fix is released with [elektra 0.9.2](https://www.libelektra.org/news/0.9.2-release).
